charts/airbyte/values.yaml

```diff
@@ -1237,7 +1237,7 @@ minio:
 
 ## @section cron parameters
 
-cron:
+airbyte-cron:
```

Comment:
Let's rename the chart so that it matches all the other, instead of deviated from the current convention